.cicd/pipeline.yml

```diff
@@ -1,4 +1,5 @@
 steps:
+
   - wait
 
   - trigger: "eosio-dot-contracts-base-images-beta"
```

Comment:
Norsegaud removed the base images pipeline in favor of building base images in the main pipeline, so we will have to do the same in a future pull request to consider this pipeline the same.